strings/loot_data/all_spawn_data.txt

```diff
@@ -6199,6 +6200,7 @@ paths    spawn_tags    blacklisted    spawn_value    spawn_price    prob_accompa
 /obj/item/weapon/gun_upgrade/mechanism/overshooter    obj;item;gun_upgrade;guns    blacklisted=0    spawn_value=4.54545   spawn_price=100   prob_accompanying_obj=100    accompanying_objs=nothing
 /obj/item/weapon/gun_upgrade/mechanism/glass_widow    obj;item;gun_upgrade;guns    blacklisted=1    spawn_value=1.92308   spawn_price=100   prob_accompanying_obj=100    accompanying_objs=nothing
 /obj/item/weapon/gun_upgrade/mechanism/weintraub    obj;item;gun_upgrade;guns    blacklisted=0    spawn_value=3.125   spawn_price=100   prob_accompanying_obj=100    accompanying_objs=nothing
+/obj/item/weapon/gun_upgrade/mechanism/funswitch    obj;item;gun_ungrade;guns    blacklisted=0    spawn_value=3.125   spawn_price=100   prob_accompanying_obj=100    accompanying_objs=nothing 
```

Comment:
This is an automatically generated file and doesn't need to be changed for this PR.